### PR TITLE
MCLOUD-7204: Update Redis and ES service versions

### PR DESCRIPTION
**Pull request by bdenham**
## Purpose of this pull request

This pull request (PR) updates the Redis and Elasticsearch services versions.

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/project/services-redis.html
- https://devdocs.magento.com/cloud/project/services.html#service-versions
- https://devdocs.magento.com/cloud/docker/docker-containers.html#service-configuration-options

### CODE REVIEW

File: src/cloud/docker/docker-containers.md [hunk 1]
```diff
@@ -44,13 +44,13 @@ The following table shows the options to customize service container configurati
...
-| [elasticsearch] | Elasticsearch | `--es`<br>`--es-env-var`<br>`--no-es` | 1.7, 2.4, 5.2, 6.5, 6.8, 7.5, 7.6, 7.7 | Use the options to specify the Elasticsearch version,  customize Elasticsearch configuration options, or to build a Docker environment without Elasticsearch.
+| [elasticsearch] | Elasticsearch | `--es`<br>`--es-env-var`<br>`--no-es` | 1.7, 2.4, 5.2, 6.5, 6.8, 7.5, 7.6, 7.7, 7.9 | Use the options to specify the Elasticsearch version,  customize Elasticsearch configuration options, or to build a Docker environment without Elasticsearch.
 | [fpm][fpm-container] | PHP FPM | `--php`<br>`--with-xdebug` | 7.0, 7.1, 7.2, 7.3, 7.4 |  Used for all incoming requests. Optionally, install a specific php version or add Xdebug to debug PHP code in the Docker environment.
```
Comment (oshmyheliuk) [inline src/cloud/docker/docker-containers.md:48]: Not related to changes in this pr, but cloud docker version 1.2+ does not have 7.0 and 7.1 PHP images

Comment (bdenham) [reply src/cloud/docker/docker-containers.md:48]: Thanks @oshmyheliuk, I'll remove those.

File: src/cloud/project/services-redis.md [hunk 2]
```diff
@@ -5,12 +5,12 @@ functional_areas:
   - Cloud
   - Setup
 redirect_from:
-  - /cloud/project/project-conf-files_services-redis.html 
+  - /cloud/project/project-conf-files_services-redis.html
 ---
 
 [Redis](http://redis.io) is an optional, backend cache solution that replaces the Zend Framework [Zend_Cache_Backend_File](http://framework.zend.com/apidoc/1.0/Zend_Cache/Backend/Zend_Cache_Backend_File.html), which is used in Magento 2 by default.
 
-We support Redis versions 3.2 and 5.0. See [Configure Redis]({{ site.baseurl }}/guides/v2.3/config-guide/redis/config-redis.html).
```
Comment (andriyShevtsov) [inline src/cloud/project/services-redis.md:13]: Here we write that we support Redis 3.2, 5.0, and 6.0. but on page https://devdocs.magento.com/cloud/project/services.html#service-versions exist version 4.0. It doesn't look clear for me.

Comment (bdenham) [reply src/cloud/project/services-redis.md:13]: @andriyShevtsov Good catch. From here https://wiki.corp.magento.com/display/PRODM/Platform+Health+Roadmap it looks like 4.0 should be removed. So I will remove 4.0 from the service version table. If this is not correct, let me know.

Comment (bdenham) [reply src/cloud/project/services-redis.md:13]: @oshmyheliuk and @andriyShevtsov: Redis 4.0 is also included in the docker container table, as shown here: https://devdocs.magento.com/cloud/docker/docker-containers.html#service-configuration-options.

So do I just need to add 4.0 to the `services-redis` page? It's unclear to me.

**Review by hguthrie — APPROVED** [review]

**Comment by bdenham** [comment]
running tests

**Comment by bdenham** [comment]
running tests

**Pull request merged**